src/actions/bots.test.js

```diff
@@ -0,0 +1,132 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import assert from 'assert';
+import nock from 'nock';
+
+import * as BotActions from 'actions/bots';
+import {Client4} from 'client';
+
+import TestHelper from 'test/test_helper';
+import configureStore from 'test/test_store';
+
+describe('Actions.Bots', () => {
+    let store;
+    beforeAll(async () => {
+        await TestHelper.initBasic(Client4);
+    });
+
+    beforeEach(async () => {
+        store = await configureStore();
+    });
+
+    afterAll(async () => {
+        await TestHelper.tearDown();
+    });
+
+    it('loadBots', async () => {
+        const bots = [TestHelper.fakeBot(), TestHelper.fakeBot()];
+        nock(Client4.getBotsRoute()).
+            get('').
+            query(true).
+            reply(201, bots);
+
+        await BotActions.loadBots()(store.dispatch, store.getState);
```

Comment:
We should actually be calling `dispatch` on the result of these action creators.

```javascript
await store.dispatch(BotActions.loadBots());
```

Both ways of calling this work the same with async actions, but the way you're doing it doesn't work with plain redux actions. Also, doing it this way saves having to pass `getState` explicitly

Reply:
Fixed.